pom.xml

```diff
@@ -155,6 +183,18 @@
     </build>
     <dependencies>
         <dependency>
+            <groupId>org.jboss.modules</groupId>
+            <artifactId>jboss-modules-jdk9-supplement</artifactId>
+            <version>1.0.Final</version>
```

Comment:
That's OK, tags are easy to do.  I expected that we may have lots of releases at first.